CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [UNRELEASED] - YYYY-MM-DD
 ### Fixed
 - [#99](https://github.com/equinor/webviz-core-components/pull/99) - Fixed bug which prevented using the download button in `WebvizPluginPlaceholder` and started to download when component was mounting. 
+- [#105](https://github.com/equinor/webviz-core-components/pull/105) - Fixed bug when updating Select values from a Dash callback.
 
 ### Added
 - [#96](https://github.com/equinor/webviz-core-components/pull/96) - Added publishing of npm package to Github Workflow
```

Comment:
The `[UNRELEASED]` is now released, update with `0.3.0` and create new `[UNRELEASED]`?